odrive:
  axis:
    left: 0
    right: 1
  config:
    current_lim: 4
    watchdog_timeout: 0.1
  ids:
    front: "335D36623539"
    middle: "335B36563539"
    back: "2066377F5753"
  multiplier:
    vel_cmd_multiplier_left: 50
    vel_cmd_multiplier_right: -50
    vel_est_multiplier_left: -0.02513
    vel_est_multiplier_right: 0.02513
